src/server/cluster/cluster_family.cc

```diff
@@ -1002,6 +1005,41 @@ void ClusterFamily::DflyMigrateAck(CmdArgList args, SinkReplyBuilder* builder) {
   return builder->SendLong(attempt);
 }
 
+void ClusterFamily::BreakStalledFlowsInShard() {
```

Comment:
where do you call this function I dont see...
Is it relay per shard? from what I ready here it looks like global server migration and not shard migration process